runtime/parachains/src/disputes.rs

```diff
@@ -831,30 +778,10 @@ impl<T: Config> Pallet<T> {
 				<Disputes<T>>::insert(session_index, candidate_hash, &dispute);
 
 				if <Included<T>>::contains_key(&session_index, &candidate_hash) {
```

Comment:
I think this if needs to go and we should take the weight from it.

Reply:
Okay, I'll remove it, but for my own benefit - could you please elaborate on why?

Reply:
Having an if skipping the spamslot handling via continue does not make sense, if there is no spam slot handling. But we should use the weight as if the if were true.

Reply:
(because that's the weight without updating spam slots)

Reply:
Yes, very good catch.